content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -176,23 +182,23 @@ On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the
 product configured in the steps above. After updating any other necessary information,
 :guilabel:`Confirm` the quotation.
 
-.. note::
-   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at this
-   time. That is because no services have been performed; in other words nothing has been delivered,
-   therefore, there is nothing to invoice.
+.. important::
+   Unlike with the prepaid services quotation, Odoo does **not** allow an invoice to be created at
+   this time. That is because no services have been performed; in other words nothing has been
+   delivered, therefore, there is nothing to invoice.
 
 Create a helpdesk ticket for time-tracked services
 --------------------------------------------------
 
 To record a *Timesheet* entry for time-tracker services, go to :menuselection:`Helpdesk` and select
 the appropriate team for which these services apply.
 
-If there is already an existing ticket for this issue, select it from the kanban view. This will
-open the ticket details form. If there is no existing ticket for this customer issue, click
+If there is already an existing ticket for this issue, select it from the Kanban view. This opens
+the ticket details form. If there is no existing ticket for this customer issue, click
 :guilabel:`New` to create a new ticket and enter the necessary customer information on the blank
 ticket details form.
```

Comment:
I think you wrote a doc about making a new helpdesk ticket? It could be linked to the 'create a new ticket' part here, and also above where it says the same thing. Not necessary, just an add-on if you wanted.